apps/vmq_enhanced_auth/src/vmq_enhanced_auth.erl

```diff
@@ -209,6 +210,18 @@ init() ->
         ?TABLES
     ).
 
+set_acl_version_metrics(File) ->
+    case vmq_util:extract_version(File) of
+        Version when is_list(Version) ->
+            vmq_metrics:update_config_version_metric({acl_version, [{version, Version}]});
```

Comment:
Why can't we pass only Version as arg?

Reply:
Updated the fn to take Metric name and Version as arguments.